scripts/hostapd/install

```diff
@@ -27,10 +27,10 @@ echo "iface lo inet loopback" | sudo tee --append /etc/network/interfaces > /dev
 echo "" | sudo tee --append /etc/network/interfaces > /dev/null
 echo "allow-hotplug wlan0" | sudo tee --append /etc/network/interfaces > /dev/null
 echo "iface wlan0 inet static" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    address 10.0.0.1" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    netmask 255.255.255.0" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    network 10.0.0.0" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    broadcast 10.0.0.255" | sudo tee --append /etc/network/interfaces > /dev/null
+echo "    address 10.10.0.1" | sudo tee --append /etc/network/interfaces > /dev/null
```

Comment:
What is the reason for shifting to a new subnet?

Reply:
So we can have `10.11.0.0/16` for VPN?

Reply:
So first the config change IS correct and it WILL work

However i will point out that the change is NOT NEEDED and here is why:

The existing config  is already "10.0.0.0" netmask "255.255.255.0" (or 10.0.0.0/24)
This means the  range is 10.0.0.0 - 10.0.0.255

It does not overlap with 10.10.0.0/24 who's  range is 10.10.0.0-10.10.0.255
So you don't really need to make this change and still have VPN on 10.10.0.0

Why this whole thing happened:
If  the netmask was 255.0.0.0 for either ranges it would cause a problem because the range would be 10.0.0.0 - 10.255.255.255 and hence overlap each other

I think what happened is when you where testing it at first you assigned the ip as such
```ifconfig tun0 10.10.0.1```
instead of
```ifconfig tun0 10.10.0.1/24```

In the 10.x.x.x is defined as a CLASS A network, and as such it defaults to a netmaks of 255.0.0.0.  

This means that 10.0.0.0 and 10.10.0.0 overlapped and  caused problems

When we talked i didn't not realize what happend and did not have the config in front of me, assumed that 10.0.0.0 was set to 255.0.0.0 as well (it was not)

Reply:
TL;DR - The ranges actually never overlap with the old config.
10.0.0.0/24  10.10.0.0/24

Reply:
Ahhhhh yes I saw `10.0.0.0` and just assumed it was a `255.0.0.0`. Now I remember that because we know the Raspberry Pi cannot handle so many clients anyways, we made it a `255.255.255.0`, and DHCP assign only 128 of those (which still is excessive).

Reply:
I know that this addition has been added, and that this may be a dumb question, but is there any specific type of VPN that this will be able to work with (like github.com/pivpn/pivpn or OpenVPN).

The reason I ask is that since cjdns is P2P, I assumed that a VPN would mess up some things...

Reply:
Kind of depends what your trying to do.

NON-CJDNS  traffic cannot exist on CJDNS 
CJDN's IPTUNNEL basically creates a VPN through CJDNS's network to allow non-cjdns traffic through. This is how you can "share" normal internet access with peers on CJDNS/Hyperborea

You can still run OpenVPN  or whatnot on your node.  The trick would be to to engineer your traffic to actually do something.